src/rollup/types.d.ts

```diff
@@ -363,13 +363,17 @@ export interface RenderedModule {
 }
 
 export interface RenderedChunk {
+	dynamicImports: string[];
+	exports: string[];
+	facadeModuleId: string | null;
```

Comment:
Would `isFacade` not handle this same information, meaning `facadeModuleId === imports[0]`?

Reply:
Not necessarily. Having a `facadeModuleId` does not mean this is a specially created facade but that this chunk has the same interface and purpose as the given module. When only having a single entry and no dynamic imports, the `facadeModuleId` of the created chunk would be the entry point id. If this not properly reflect this meaning, maybe another name would be better?